examples/units_and_coordinates/AltAz Coordinate Transfer Final.py

```diff
@@ -0,0 +1,46 @@
+"""
+============================================
+Helioprojective/AltAz Coordinate Conversions
+============================================
+
+This is a very simple example to show how to convert
+Helioprojective Coordiantes to Altitude/Azimuth Coordinates.
+In this example, we use balloons observations to include
+elevation as a factor.
+"""
+from astropy.coordinates import EarthLocation, AltAz, SkyCoord
+from astropy.time import Time
+from sunpy.coordinates import frames, get_sunearth_distance
+import astropy.units as u
+
+######################################################################################
+# We use AstroPy SkyCoords to define the center of the Sun
```

Comment:
If you can change, AstroPy SkyCoords to AstroPy's `~astropy.coordinates.SkyCoord`. I forgot how to escape characters in markdown, but around ~astropy.coordinates.SkyCoord, you want to add a back backtick at the start and the end. This way it should  create a link to Astropy's Documentation.

Reply:
That's really convenient, thanks!